pkg/api/validation/validation.go

```diff
@@ -1975,6 +1975,9 @@ func validateContainers(containers []api.Container, volumes sets.String, fldPath
 		if len(ctr.Image) == 0 {
 			allErrs = append(allErrs, field.Required(idxPath.Child("image"), ""))
 		}
+		if len(strings.TrimSpace(ctr.Image)) != len(ctr.Image) {
```

Comment:
right, anyone that calls ValidatePodSpec, so this is picked up by all the job/rc/...